woocommerce-services.php

```diff
@@ -256,6 +256,9 @@ class WC_Connect_Loader {
 
 		public const MIGRATION_DISMISSAL_COOKIE_KEY = 'wcst-wcshipping-migration-dismissed';
 
+		public const PLUGIN_NAME_FOR_LEGACY_SITES        = 'WooCommerce Tax (previously WooCommerce Shipping & Tax)';
+		public const PLUGIN_DESCRIPTION_FOR_LEGACY_SITES = 'Hosted services for WooCommerce: automated tax calculation, shipping label printing, and smoother payment setup.';
```

Comment:
It's years ago that I looked it up, but I believe the plugin's php header name and description will be translated if they exist somewhere in the pot file. Should these perhaps be wrapped in `__()` to match that behaviour? 

Note: AFAIR we can't wrap constant values in a function, but It looks like we only use them in one place, so moving them to `maybe_rename_plugin()` should allow us to add the translation functions

Reply:
good catch!! Yeah, I'll update it.